pkgs/ned1313/terrahash/registry.yaml

```diff
@@ -9,6 +9,7 @@ packages:
         asset: terrahash_{{.OS}}_{{.Arch}}.{{.Format}}
         format: tar.gz
         replacements:
+          aarch64: arm64
```

Comment:
Keys of replacements must be [Go's runtime.GOOS and runtime.GOARCH](https://pkg.go.dev/runtime#pkg-constants), so `aarch64` is invalid.

> enable installation on aarch64 (MacBook Pros M* based machines for example).

This change is unnecessary. You can install terrahash on aarch64.